tests/integration_tests/validation/conformance_suite_configs.py

```diff
@@ -25,7 +27,9 @@
 ALL_CONFORMANCE_SUITE_CONFIGS: tuple[ConformanceSuiteConfig, ...] = (
     efm_current,
     esef_ixbrl_2021,
+    esef_ixbrl_2022,
```

Comment:
Do we have an idea for how long we want to retain/run the 2021 conformance suite? For the recent EFM update, we just updated in place. Is there a reason we wouldn't want to do that here?

Reply:
We will most likely retain the 2021 suite until ESEF 2021 and older taxonomies are all deprecated and no longer accepted by authorities.